src/common/filesystembase.cpp

```diff
@@ -161,16 +161,14 @@ bool FileSystem::rename(const QString &originFileName,
     bool success = false;
     QString error;
 #ifdef Q_OS_WIN
-    const QString orig = longWinPath(originFileName);
+    const QString originalFileNameLong = longWinPath(originFileName);
     const QString dest = longWinPath(destinationFileName);
```

Comment:
You may want to implement a consistent pattern (the previous variable is called `...Long` whereas this one isn't although you call that function on both anyway).